sdk/storage/storage-blob/tsconfig.json

```diff
@@ -1,26 +1,10 @@
 {
+  "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    "alwaysStrict": true,
-    "noImplicitAny": true,
-    "preserveConstEnums": true,
-    "sourceMap": true,
-    "inlineSources": true,
-    "newLine": "LF",
-    "target": "es5",
```

Comment:
I thought storage packages (along with the core things they depend on) actually *do* need to target es5? 😅

Reply:
Yes it is safer to target es5 now for Stoarge. Though it's not a big issue if we still have opentelemetry targeting ES6 thus we'd always need to workaround it for IE11 anyway.

Reply:
Alright, I'll set es5 for storage! Thank you!